src/sidebar/media-embedder.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 /**
+* Return an HTML5 audio player with the given src URL.
+*/
+
+function audioElement(src) {
+  var html5audio = document.createElement('audio');
+  html5audio.id       = 'audio-player';
```

Comment:
I don't recall, honestly. I think I was just giving the element and ID attribute in case someone might want to style it with CSS. On second glance, I agree it can be removed.